recipes/python/assets/create_oracle_asset.py

```diff
@@ -0,0 +1,247 @@
+# Cred? if doesn't already exist
+# register creds
+
+## The script can be run with Python 3.5 or higher version.
+## The script requires 'requests' library to make the API calls. The library can be installed using the command: pip install requests.
+
+import argparse
+import requests
+import time
+import os
+import json
+
+content_type = "application/vnd.netbackup+json;version=3.0"
+
+parser = argparse.ArgumentParser(description="MSSQL Instance backup and Database restore scenario")
+parser.add_argument('-nbserver', required=True, help="NetBackup primary server name")
+parser.add_argument('-username', required=True, help="NetBackup primary server username")
+parser.add_argument('-password', required=True, help="NetBackup primary server password")
+parser.add_argument('-domainName', required=False, help="NetBackup primary server login domain name")
+parser.add_argument('-domainType', required=False, help="NetBackup primary server login domain type")
+parser.add_argument('-dbName', required=False, default="SampleDatabaseName", help="Oracle Database name to create an asset for")
+parser.add_argument('-addRMANCatalog', required=False, default=0, choices=[0,1] help="If an RMAN catalog asset should be created")

```

Comment:
Use a Boolean flag for this instead of requiring integer arguments. Also, there's a missing comma before the `help` argument.

```suggestion
parser.add_argument('-addRMANCatalog', required=False, action='store_true', help="If an RMAN catalog asset should be created")
```